### PR TITLE
feat: show lock stats

### LINKED ISSUE #109: Display Current APR on the stats page

### PR DESCRIPTION
**Pull request by JustinCast**
# Show Lock stats
_It is needed to present relevant information to the final user. Data like: apr, tvl_
## How should this be tested?
1. `yarn start`
## Notes or observations
_no observations for this PR_
## Linked issues
closes #109

### CODE REVIEW

**Comment by JustinCast** [comment]
@kesar 👋
Do we need to remove the previous rewards code?

**Comment by kesar** [comment]
> @kesar 👋
> 
> Do we need to remove the previous rewards code?
> 
> Also, it seems that the checkpointToken function is only callable by the owner

yes

**Comment by kesar** [comment]
Captura de pantalla 2021-08-17 a las 14 07 10

Still not showing anything

Please make sure you spend time testing different scenarios so we avoid this back-and-forth 🙏🏻 

if you dont have an account in mainnet w hiiq, just simulate that you have in any account you can manage or I can try to setup one for you

**Comment by JustinCast** [comment]
> Captura de pantalla 2021-08-17 a las 14 07 10
> 
> Still not showing anything
> 
> Please make sure you spend time testing different scenarios so we avoid this back-and-forth 🙏🏻
> 
> if you dont have an account in mainnet w hiiq, just simulate that you have in any account you can manage or I can try to setup one for you

Yeah sorry, I don't have an account with HiIQ to test it on mainnet. It is working on the testnet, that why I didn't notice. And since it seems to be gas related, we won't be able to notice the error using tenderly. I just set a hardcoded value (800 000)

**Comment by kesar** [comment]
> And since it seems to be gas related

But I am not performing any action, not sure why something gas related can happen

File: src/config.js [hunk 1]
```diff
@@ -15,6 +15,7 @@ module.exports = {
   pIQAddress: process.env.REACT_APP_PIQ_ADDRESS,
   pMinterAddress: process.env.REACT_APP_PMINTER_ADDRESS,
   hiIQRewardsAddress: process.env.REACT_APP_HIIQREWARDS_ADDRESS,
+  feeDistributorAddress: process.env.REACT_APP_FEE_DISTRIBUTOR_ADDRESS,
```
Comment (kesar) [inline src/config.js:18]: this needs to be in the env template file

**Comment by kesar** [comment]
https://github.com/EveripediaNetwork/iq-bridge-ui/issues/109#issuecomment-896841577
> I think we should show next distribution window time as well

I dont see this implemented

**Comment by kesar** [comment]
Captura de pantalla 2021-08-18 a las 14 41 27

**Pull request merged**